plugins/source-terasology-modules/gatsby-node.js

```diff
@@ -0,0 +1,169 @@
+const { graphql } = require("@octokit/graphql");
+const { DateTime } = require("luxon");
+
+const PLUGIN_NAME = "source-terasology-modules";
+
+const query = `
+query Modules($cursor:String) {
+  organization(login: "Terasology") {
+    repositories(first: 20, after: $cursor) {
+      nodes {
+        id
+        name
+        url
+        description
+        homepageUrl
+        openGraphImageUrl
+        usesCustomOpenGraphImage
+
+        moduleTxt: object(expression: "develop:module.txt") {
+          ...on Blob {
+            text
+          }
+        }
+
+        readme: object(expression: "develop:README.md") {
+          ...on Blob {
+            text
+          }
+        }
```

Comment:
did I miss it or do we not use this anywhere yet?

Reply:
See https://github.com/MovingBlocks/ModuleSite/pull/179#pullrequestreview-1238074972